credential_adapter/credential_adapter.go

```diff
@@ -81,18 +87,18 @@ func (c *credentialAdapterClient) CreateIssuanceChallenge(cred_type string, did
 	return IssuanceChallenge{Url: resp.Endpoint, CredType: cred_type, Nonce: resp.Nonce}, nil
 }
 
-func (c *credentialAdapterClient) CreatePresentationChallenge(cred_type string) (IssuanceChallenge, error) {
+func (c *credentialAdapterClient) CreatePresentationChallenge() (*PresentationChallenge, error) {
 	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 	defer cancel()
 	resp, err := c.client.CreatePresentationChallenge(ctx, &AdapterApi.CreatePresentationChallengeRequest{
-		CredentialType: getCredentialEnumFromName(cred_type),
+		CredentialType: AdapterApi.CredentialType_CREDENTIAL_TYPE_UNSPECIFIED,
```

Comment:
Looks like unconventional name to me

Reply:
@sKudryashov Which part exactly?

This part `AdapterApi.CredentialType_CREDENTIAL_TYPE_UNSPECIFIED`?